src/ant/README.md

```diff
@@ -0,0 +1,48 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations based on an input LEF and DEF file, a report
+will be generated to indicate violated nets. 4 APIs are provided as an interface with FastRoute to preemptively fix antenna violation as well as used for the diode insertion flow:
+
+## Antenna Check Commands
+
+---
+
+ - `load_antenna_rules`: import antenna rules to ARC, must be called before other commands
+ - `check_antennas`: check antenna violations on all nets and generate a report
+   - -path: specify the path to save the report
+   - -simple_report: provides a summary of the violated nets 
+
+## Antenna Report Example
+---
+
+This is an example of the detailed and simple reports of the antenna checker:
+
+| <img src="./doc/images/ant_report_print.png" width=400px> |  
+|:--:|
+
+Abbreviations Index:
+  - `PAR`: Partial Area Ratio
+  - `CAR`: Cumulative Area Ratio
+  - `Area`: Gate Area
+  - `S. Area`: Side Diffusion Area
+  - `C. Area`: Cumulative Gate Area
+  - `C. S. Area`: Cumulative Side (Diffusion) Area
+
+## Antenna Repair Commands
```

Comment:
empty section?

Reply:
removed. It used to have the APIs in C++